code/modules/surgery/helpers.dm

```diff
@@ -147,12 +147,13 @@
 	var/turf/T = get_turf(M)
 	if(locate(/obj/machinery/optable, T))
 		return 1
-	else if(locate(/obj/structure/table, T))
+	if(locate(/obj/structure/bed/roller/holo, T))
+		return 0.9
+	if(locate(/obj/structure/table, T) || locate(/obj/structure/bed/roller, T))
 		return 0.8
```

Comment:
Would it make sense balance wise to bump normal roller beds to 0.9 as well? 
After all tables are ubiquitous, while roller beds are a lot rarer.

Reply:
This was the order one of the balance team members requested. I'm not opposed to changing it but I'd need some of them to ask or be okay with it